examples/micro-http-server-ts/src/logger.ts

```diff
@@ -0,0 +1,57 @@
+import * as store from '@leapfrogtechnology/async-store';
+
+/**
+ * Get unique request id from store or return empty string.
+ *
+ * @returns {string}
+ */
+function getRequestId() {
+  return (store.getId() || '').substring(0, 8);
```

Comment:
The new version has a new method.
```suggestion
  return store.getShortId();
```